site/content/en/docs/commands/podman-env.md

```diff
@@ -4,6 +4,14 @@ description: >
   Configure environment to use minikube's Podman service
 ---
 
+## Requirements
+
+- **Podman version 4.9.2 or newer is required.**
+- Support for Podman v3 and varlink-based communication has been removed. The `podman-env` command now configures your environment to use the Podman REST API socket, as required by Podman v4+.
+
+{{% pageinfo color="warning" %}}
+**Note:** If you are using an older version of Podman, please upgrade to at least v4.9.2 to use `minikube podman-env`. Legacy varlink-based workflows are no longer supported.
```

Comment:
The version 4.9.2 is quite specific, it might as well say 4.x (which is not supported anyway) and later (5.x)

https://endoflife.date/podman

Reply:
I don't think 4.9.3 is random*, but upgrading the client is not an issue.
Some people have problems with _downgrading_, but that's different.

* it is the "last" common v4 version, just like 1.9.3 was the last v1 version.
We could upgrade to v5, but it is not available in every distribution yet.

Reply:
There are no features in podman 5 (or even in podman 4, but anyway)
that are required by cri-o, as far as I know? Main issue is client compat.